Sources/ScrollingTabView.swift

```diff
@@ -74,13 +76,24 @@ public class ScrollingTabView: UIView {
             switch tabSizing {
             case .fitViewFrameWidth:
                 calculateItemSizeToFitWidth(frame.width)
+                if let layout = collectionView.collectionViewLayout as? ScrollingTabViewFlowLayout {
```

Comment:
could we do this in one place and set to try in the one exception? might even work with a guard up front